File: despatch.WEB/app/src/swagger/models/user-addresses-request.ts
/* tslint:disable */
import { BaseEntityRequest } from './base-entity-request';
import { AddressRequest } from './address-request';
import { UserRequest } from './user-request';
export interface UserAddressesRequest extends BaseEntityRequest{
  address?: AddressRequest;
  addressId?: string;
  user?: UserRequest;
  userId?: string;
}

File: despatch.Domain/Models/Default/Request/UserAddressesRequest.cs
namespace despatch.Domain.Models.Default.Request
{
    public class UserAddressesRequest : BaseEntityRequest
    {
        public string UserId { get; set; }
        public string AddressId { get; set; }

        public virtual UserRequest User { get; set; }
        public virtual AddressRequest Address { get; set; }
    }
}
